### DISCUSSION TITLE
Exclude own actions from Slack notifications

### DISCUSSION THREAD

**Discussion by sewi-cpan** [community/community#65869] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

The ["scheduled reminders" include "real-time alert"](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/managing-your-scheduled-reminders)
![image](https://user-images.githubusercontent.com/2063104/220056108-23e4d335-7f4d-4bf9-aab2-3f85f3060fd8.png)
They speed up teamwork a lot. The one thing missing is an option to **exclude your own actions**.

No real-time messages should be created for the recipient's own actions.

**Comment by mohammadmehrani** [comment]
Where should I make these settings?
